SwiftCompilerSources/Sources/Basic/Utils.swift

```diff
@@ -60,6 +61,13 @@ extension String {
       return c(BridgedStringRef(data: buffer.baseAddress, length: buffer.count))
     }
   }
+
+  /// Underscored to avoid name collision with the std overlay.
+  /// To be replaced with an overlay call once the CI uses SDKs built with Swift 5.7.
+  public init(_cxxString s: std.string) {
+    self.init(cString: s.c_str())
```

Comment:
Is it not required anymore that `s` is a var for calling `c_str`? (I can't remember why we had to do this).

Reply:
We were using `var` to avoid breaking the hosttools build for people using Xcode 13.2 or older. Now that Xcode 13.4 is released, this is probably no longer necessary.
We could produce a friendly CMake error for an outdated Swift compiler version if we merge https://github.com/apple/swift/pull/59020.